places/berlin.ini

```diff
@@ -111,3 +111,28 @@ url_path = div/div/div[2]/h2/a/@href
 title_path = div/div/div[2]/h2/a
 location_path = div/div/div[2]/address:(?<=,)[^,]*
 rooms_path = div/div/div[2]/ul/li[1]/span[2]
+
+[company:www.blp-management.com]
+url = https://blp-management.com/immobilien/?post_type=immomakler_object&vermarktungsart=miete&nutzungsart=wohnen&typ=wohnung&ort=berlin&center&objekt-id&collapse=in
+ad_path = .//div[@class='property-container']
+url_path = div[2]/h3/a/@href
+title_path = div[2]/h3/a
+location_path = div[2]/div
+rooms_path = div[2]/div[2]/div[2]
+
+[company:www.immonexxt.com]
+url = https://www.immonexxt.com/de/mietangebote-wohnen#filter=.geo-ort-berlin
+ad_path = .//div[@id='estate_list']/a
+url_path = ./@href
+title_path = div/div[2]/h2
+location_path = div/div[2]/ul/li[2]/div[2]/span[1]
```

Comment:
Because the `geo-ort-berlin` filter in the URL only works with JavaScript, we receive ads for Potsdam here. But this could be fixed by changing the path so that it includes the city name and then adding a `location_filter` for `Berlin`. (see e.g. Agas Immobilien)